File: Material/TransitSoft/TransitSoftPersistance/DAO/CamaraDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using TransitSoft.Model;

namespace TransitSoft.DAO
{
    public interface CamaraDAO
    {
        int? Insertar(Camara camara);
        int? Modificar(Camara camara);
        int? Eliminar(int id);
        Camara ObtenerPorId(int id);
        IList<Camara> ListarTodos();
    }
}

File: EX2/EX2/Material/TransitSoft_ConexionMultiple/TransitSoftPersistencia/src/main/java/pe/edu/pucp/transitsoft/dao/CamaraDAO.java

package pe.edu.pucp.transitsoft.dao;

import java.util.List;
import pe.edu.pucp.transitsoft.modelo.Camara;

/**
 *
 * 
 */
public interface CamaraDAO {
    public Integer insertar(Camara camara);
    public Integer modificar(Camara camara) ;
    public Integer eliminar(int id);
    public Camara obtenerPorId(int id);
    public List<Camara> listarTodos();
}
